zipline/data/benchmarks.py

```diff
@@ -24,5 +25,14 @@ def get_benchmark_returns(symbol, start_date, end_date):
     start_date is **not** included because we need the close from day N - 1 to
     compute the returns for day N.
     """
-    df = web.DataReader(symbol, 'google', start_date, end_date)
-    return df["Close"].sort_index().tz_localize('UTC').pct_change(1).iloc[1:]
+    df = web.DataReader(symbol, 'google', start_date, end_date) 
+    df.index = df.index.tz_localize('UTC')
+
+    calendar = get_calendar("NYSE")
+    start_index = calendar.all_sessions.searchsorted(start_date)
```

Comment:
Could you please explain why indexing is required here? The original code seemed to work well with start_date and end_date supplied to DataReader.